examples/spring/src/main/resources/application.yml

```diff
@@ -3,3 +3,4 @@ graphql:
   subscriptions:
     # Send a ka message every 1000 ms (1 second)
     keepAliveInterval: 1000
+    endpoint: graphql
```

Comment:
i assume this is just from testing -> lets revert back to default

Reply:
Done